pkgs/development/python-modules/fluidasserts/default.nix

```diff
@@ -0,0 +1,142 @@
+{ buildPythonPackage
+, fetchPypi
+, isPy37
+, lib
+
+# pythonPackages
+, aiohttp
+, androguard
+, azure-identity
+, azure-keyvault-keys
+, azure-keyvault-secrets
+, bandit
+, bcrypt
+, beautifulsoup4
+, boto3
+, cfn-flip
+, cython
+, dnspython
+, colorama
+, configobj
+, defusedxml
+, GitPython
+, google_api_python_client
+, kubernetes
+, ldap3
+, mysql-connector
+, names
+, ntplib
+, oyaml
+, paramiko
+, pillow
+, psycopg2
+, pycrypto
+, pygments
+, pyjks
+, pynacl
+, pyopenssl
+, pypdf2
+, pysmb
+, python_magic
+, pytz
+, requirements-detector
+, selenium
+, tlslite-ng
+, viewstate
+
+# custom pythonPackages
+, mixpanel'
+}:
+
+buildPythonPackage rec {
+  pname = "fluidasserts";
+  version = "20.1.21324";
+  disabled = !isPy37;
+
+  src = fetchPypi {
+    inherit pname version;
+    format = "setuptools";
+    extension = "zip";
+    sha256 = "061jhd9s3mikzld8240r121clcdxsyniqbjqcripii387khq4jxz";
+  };
+
+  patchPhase = ''
+    # Version mismatches between current FluidAsserts and Nixpkgs
+    substituteInPlace ./setup.py \
+      --replace 'pycrypto==2.6.1' 'pycrypto==3.9.4' \
+      --replace 'tlslite-ng==0.8.0-alpha29' 'tlslite-ng==0.7.5' \
+      --replace 'boto3==1.10.17' 'boto3==1.10.1' \
+      --replace 'cfn-flip==1.2.2' 'cfn-flip==1.1.0.post1' \
+
+    # Functionality that will be not present for the momment
+    #   but that we'll work to add in the future
+    substituteInPlace ./setup.py \
+      --replace "'azure==4.0.0'," "" \
```

Comment:
please upstream that the azure package should not be installed directly. You should just install the individual namespace packages that you actually need.

Reply:
I'll give it a look, for the moment I think we are find without that little sub-portion of fluidasserts
https://gitlab.com/fluidattacks/asserts/issues/872

Reply:
thanks ! It works like a charm

once I get the change I'll update the PR

Reply:
wow, that was fast